src/mbgl/map/tile_parser.hpp

```diff
@@ -48,11 +46,11 @@ class TileParser : private util::noncopyable
 public:
     void parse();
 
-private:
+protected:
```

Comment:
You got one of them, but this one's still present. Also the added `virtual` below can be reverted.